src/java/org/openzal/zal/Domain.java

```diff
@@ -166,9 +162,10 @@ public String getPasswordChangeListener()
     }
   }
 
-  protected <T> T toZimbra(@NotNull Class<T> cls)
```

Comment:
Revert this line, the cast is typless to avoid ever importing Domain inside the library user.

Reply:
I modify it to remove the duplication of code, this method was invoke 11 times and all with the same parameter.
If you want, I can revert the method and add a new method with the same name and with my changes.
If this method can be used inside a library user, why make it protected?